### PR TITLE
CLOUDP-122260: Update atlascli cluster commands description

### PR DESCRIPTION
**Pull request by andreaangiolillo**
## Proposed changes

_Jira ticket:_ CLOUDP-122260

## Checklist

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added any necessary documentation in document requirements section listed in [CONTRIBUTING.md](https://github.com/mongodb/mongocli/blob/master/CONTRIBUTING.md) (if appropriate)
- [x] I have addressed the @mongodb/docs-cloud-team comments (if appropriate)
- [x] I have updated [e2e/E2E-TESTS.md](https://github.com/mongodb/mongocli/blob/master/e2e/E2E-TESTS.md) (if an e2e test has been added)
- [x] I have run `make fmt` and formatted my code

## Further comments
**Issue:** Some atlascli cluster commands still have mongocli in their descriptions.

**Fix:**

```
./bin/atlas cluster create --help
To get started quickly, specify a name for your cluster, a cloud provider, and a region to deploy a three-member replica set with the latest MongoDB server version.
For full control of your deployment, or to create multi-cloud clusters, provide a JSON configuration file with the --file flag.

Usage:
  atlas clusters create [name] [flags]

Examples:

  Deploy a three-member replica set in AWS:
  $ atlas cluster create <clusterName> --projectId <projectId> --provider AWS --region US_EAST_1 --members 3 --tier M10 --mdbVersion 5.0 --diskSizeGB 10

  Deploy a three-member replica set in AZURE:
  $ atlas cluster create <clusterName> --projectId <projectId> --provider AZURE --region US_EAST_2 --members 3 --tier M10  --mdbVersion 5.0 --diskSizeGB 10

  Deploy a three-member replica set in GCP:
  $ atlas cluster create <clusterName> --projectId <projectId> --provider GCP --region EASTERN_US --members 3 --tier M10  --mdbVersion 5.0 --diskSizeGB 10

  Deploy a cluster or a multi-cloud cluster from a JSON configuration file:
  $ atlas cluster create --projectId <projectId> --file <path/to/file.json>
```

### CODE REVIEW

File: internal/cli/atlas/clusters/update.go [hunk 1]
```diff
@@ -116,18 +129,10 @@ func UpdateBuilder() *cobra.Command {
...
-
-  Update disk size for a cluster
-  $ mongocli atlas cluster update <clusterName> --projectId <projectId> --diskSizeGB 20
-
-  Update MongoDB version for a cluster
-  $ mongocli atlas cluster update <clusterName> --projectId <projectId> --mdbVersion 4.2`,
-		Args: require.MaximumNArgs(1),
+		Use:     "update [clusterName]",
+		Short:   "Update a MongoDB cluster.",
+		Example: fmt.Sprintf(exampleUpdate, exampleCmd, exampleCmd, exampleCmd),
```
Comment (blva) [inline internal/cli/atlas/clusters/update.go:134]: nit: this made me learn something https://stackoverflow.com/questions/37001449/replace-all-variables-in-sprintf-with-same-variable and maybe we can try one of the options here? e.g. `s := fmt.Sprintf("%[1]v in %[1]v is %[1]v", val)
`

**Review by blva — APPROVED** [review]
LGTM - 1 small nit

File: internal/cli/atlas/clusters/update.go (hunk 1 above)
Comment (andreaangiolillo) [reply internal/cli/atlas/clusters/update.go]: Thank you for sharing! Updated

**Review by blva — APPROVED** [review]
LGTM!

**Comment by andreaangiolillo** [comment]
No need for doc review as we are updating command examples that contain `mongocli` instead `atlas`

**Pull request merged**